src/components/Footer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div className="bg-green-700 text-white text-center py-14 text-base md:text-xl px-6">
+      <h1>Copyright © 2023 Yorkshire Woods Ground Crew. All rights reserved.</h1>
```

Comment:
issue: Generally, there should only be one h1 per page.
issue: We have a BaseHeader component, so we can use it here.